WordPress/Classes/Utility/ImmuTable.swift

```diff
@@ -279,6 +279,14 @@ public class ImmuTableViewHandler: NSObject, UITableViewDataSource, UITableViewD
         return cell
     }
 
+    public func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return viewModel.sections[section].headerText
+    }
+
+    public func tableView(tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        return viewModel.sections[section].footerText
```

Comment:
Question: Are we sure we don't wanna just wire `WPTableViewSectionHeaderFooterView` instead?. (Every spot i can think of, we use custom Headers / Footers)